README.md

```diff
@@ -21,61 +21,157 @@ e.g.
 docker push quay.io/kiegroup/kie-cloud-operator:<version>
 ```
 
-## Deploy to OpenShift 4.5+ using OLM
+## Deploy to OpenShift 4.7+ using OLM
 
-To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io account to create
+To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io (https://quay.io/) account to create
 an application repository. Follow the [authentication](https://github.com/operator-framework/operator-courier/#authentication) 
 instructions for Operator Courier to obtain an account token. 
 This token is in the form of "basic XXXXXXXXX" and both words are required for the command.
-
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. 
+ 
 Also note that the push command does not overwrite an existing repository, 
 and it needs to be deleted before a new version can be built and uploaded. 
 Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) 
 to load your operator bundle in OpenShift.
 
-**Create your own index image**
 
-Requires [opm](https://github.com/operator-framework/operator-registry/releases) v1.15.3+ -
+Requires [opm](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/) 
 
-```bash
-USERNAME=tchughesiv
-VERSION=$(go run getversion.go -csv)
-IMAGE=quay.io/${USERNAME}/rhpam-operator-bundle
-BUNDLE=${IMAGE}:${VERSION}
+Requires [podman](https://podman.io/)
+
+Requires [cekit](https://github.com/cekit/cekit)
+
+**Create your own Bundle**
+Remove the following line from deploy/olm-catalog/dev/<last-version>/manifest/businessautomation-operator.clusterserviceversion.yaml
+```console
+replaces: businessautomation-operator.<last-version>
+```
+
+Activate Cekit and run the following command
+```console
 $ make bundle-dev
+```
+
+the last log line is something like this:
+```console
+INFO  Image built and available under following tags: quay.io/<your_quay_username>/rhpam-operator-bundle:7.12.1, quay.io/${USERNAME}/rhpam-operator-bundle:latest
+```
+**Push the bundle on the container registry**
+
+VERSION=$(go run getversion.go)
+
+USERNAME=<your_quay_username>
+```console
+$ docker push quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION}
+```
 
+#### Build the index
+```console
+opm index add --bundles quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION} --tag quay.io/${USERNAME}/rhpam-operator--index:${VERSION} --permissive
```

Comment:
fix naming and remove permissive parameter
```suggestion
opm index add --bundles quay.io/${USERNAME}/rhpam-operator-bundle:${VERSION} --tag quay.io/${USERNAME}/rhpam-operator-index:${VERSION}
```
please fix indexing image naming in all places

Reply:
@jakubschwan without permissive the build fail